### ISSUE TITLE
"Build from source" sets db property that should not be set

### ISSUE THREAD

**Issue by Tanarri** [joyfullservice/msaccess-vcs-integration#215]
Hi,

I exported a access 2013 database with VCS 3.3.31. This database has not set "option" - "actual database" - "use cache format, which is compatible with 2010 or higher" (translated from me to english from german!) --> Picture of the setting:
![grafik](https://user-images.githubusercontent.com/6181166/116759043-a8515300-aa11-11eb-9cee-ae4d94dfa9ad.png)

Then I copied the exported files with a github account to another PC and did a "build from source". The newly created database has then set the option "use cache format, which is compatible with 2010 or higher". This should not happen.
If there are any issues wheather the option is set or not I don't know

Part of vcs-options.json 
  "Info": {
    "AddinVersion": "3.3.31",
    "AccessVersion": "15.0 32-bit"
  },

Both PCs has the same setting in Options - General:
![grafik](https://user-images.githubusercontent.com/6181166/116759344-65dc4600-aa12-11eb-964b-ddecbb4a65f3.png)

**Comment by joyfullservice** [comment]
I am not seeing any problem in my testing... Here is what it looks like on an English system:

![image](https://user-images.githubusercontent.com/7254992/116795433-99fb4980-aa9a-11eb-86d5-2bef9a933a40.png)

If I uncheck the box, it turns on the option for the 2007 compatibility cache.

![image](https://user-images.githubusercontent.com/7254992/116795445-b6978180-aa9a-11eb-8353-f6dcdf0de843.png)

I can export and build the database from source with or without the option checked, and it applies the correct setting from the original database source files.

Let me know if you find a way to reproduce the problem.

**Comment by Tanarri** [comment]
Hi @joyfullservice,

I reinvestigate the issue and can reproduce it and also found the reason.

It turns out that the source database, which I exported, did not export the property "Use Microsoft Access 2007 compatible cache" in the "dbs-properties.json". This option was not in the json file!
Also the options "Clear Cache on Close" and "Never Cache" were missing in the json file.

The I activated "Use Microsoft Access 2007 compatible cache" and exported again. Then the options were saved in the json file with the correct value.

As a test I deactivated the option "Use Microsoft Access 2007 compatible cache" again. The option still were in the json file with the correct value.

It seems that my database, which is an accdb, is too old?! Maybe it was created in Access before Access knew this option?

So it isn't a problem for me, but it thought it could be a bug. It isn't one.

We can close this issue I think.
